src/libcpu/src/jit/jit.cpp

```diff
@@ -99,20 +128,105 @@ initStubs()
    a.bind(exitLabel);
    a.mov(a.niaMem, a.finaleNiaArgReg);
    a.mov(asmjit::x86::rax, a.stateReg);
-   a.add(asmjit::x86::rsp, 0x38);
+   if (gJitMode == jit_mode::verify) {
+      a.mov(asmjit::x86::ecx, asmjit::X86Mem(asmjit::x86::rsp, 32, 4));
```

Comment:
Yes -- this was just more convenient since there were 4 unused bytes anyway (due to alignment stuff).